en/components/for_of.md

```diff
@@ -0,0 +1,97 @@
+---
+title: igxFor
+_description: The Ignite UI for Angular product is providing virtual directive igxFor similar to ngFor, but it is creating scrollbars and only renders small chunks of the data.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Virtualization, Performance, Virtual directive, Virtual For
+---
+
+## igxFor
+
+<p class="highlight">
+`igxForOf` is now available as an alternative to `ngForOf` for templating large amounts of data. The `igxForOf` uses virtualization technology behind the scenes to optimize DOM rendering and memory consumption. Virtualization technology works similar to Paging by slicing the data into smaller chucks which are swapped from a container viewport while the user scrolls the data horizontally/vertically. The difference with the Paging is that virtualization mimics the natural behavior of the scrollbar.
+The directive is creating scrollable containers and renders small chunks of the data. It is used inside `igx-grid` and it can be used to build virtual `igx-list`.
+`igxForOf` can be used to virtualize data horizontally, vertically or both.
+</p>
+<div class="divider"></div>
+
+### igxFor Demo
+
+<div class="sample-container loading" style="height:780px">
+    <iframe src='{environment:demosBaseUrl}/igx-for-sample-1' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<!--<button data-localize="stackblitz" class="stackblitz-btn">view on stackblitz</button> -->
+<div class="divider--half"></div>
+
+### Dependencies
+
+The `igxFor` directive is exported as as an `NgModule`, thus all you need to do in your application is to import the _IgxForOfModule_
+inside your `AppModule`:
+
+```typescript
+// app.module.ts
+
+import { IgxForOfModule } from 'igniteui-angular/main';
+
+@NgModule({
+    imports: [
+        ...
+        IgxForOfModule,
+        ...
+    ]
+})
+export class AppModule {}
+```
+
+### Usage
+
```

Comment:
@dkamburov We should add 3 sections here for horizontal, vertical and both virtualization.